torch/csrc/jit/ir/scope.cpp

```diff
@@ -124,15 +124,16 @@ std::vector<InlinedCallStackEntry> InlinedCallStack::vec() {
   std::vector<InlinedCallStackEntry> r;
   c10::optional<InlinedCallStackPtr> current = intrusive_from_this();
   while (current) {
-    r.emplace_back(std::make_tuple(
-        (*current)->fn_,
-        (*current)->source_range_,
-        (*current)->module_instance_info_));
+    r.emplace_back((*current)->getEntry());
     current = (*current)->callee_;
   }
   return r;
 }
 
+InlinedCallStackEntry InlinedCallStack::getEntry() const {
```

Comment:
Can you add some comments to this too?

Reply:
oops, these were old code that isn't needed anymore, so nice catch!